plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -143,6 +146,42 @@ class Bazel(
         )
     }
 
+    /**
+     * Collect the local path overrides from the MODULE.bazel file in [workingDir]. The overrides are collected with
+     * Buildozer and are returned as a map from package names to local path overrides. An empty map is returned if no
+     * override is defined.
+     */
+    private fun getLocalPathOverrides(workingDir: File): Map<String, String> {
+        /**
+         * Normally, it should be possible to just run "buildozer 'print module_name path'
+         * //MODULE.bazel:%local_path_override" to get the local path overrides. However, "path" is a special attribute
+         * of the "print" command and several commands must be run instead.
+         * See https://github.com/bazelbuild/buildtools/issues/1286.
+         */
+        val commandsFile = createOrtTempFile("buildozer").also {
+            it.writeText(
+                "rename path not_path|print module_name not_path|" +
+                    "rename not_path path|//MODULE.bazel:%local_path_override"
+            )
+        }
+
+        val process = ProcessCapture(
+            "buildozer",
+            "-f", commandsFile.absolutePath,
+            workingDir = workingDir,
+            environment = emptyMap()
+        )
+        require(process.exitValue in (listOf(0, 3))) {
+            "Failed to get local path overrides from 'buildozer': ${process.stderr}"
+        }
+
+        return process.stdout.lines().filter { it.isNotEmpty() }.associate { line ->
+            val (moduleName, path) = line.split(" ", limit = 2)
```

Comment:
Prefer the char-version of `split()` when splitting on single chars: `split(' ', limit = 2)`